src/test/java/spoon/test/filters/FilterTest.java

```diff
@@ -96,7 +96,7 @@ public void testFilters() throws Exception {
 		CtClass<?> foo = factory.Package().get("spoon.test.filters").getType("Foo");
 		assertEquals("Foo", foo.getSimpleName());
 		List<CtExpression<?>> expressions = foo.getElements(new RegexFilter<CtExpression<?>>(".* = .*"));
-		assertEquals(2, expressions.size());
+		assertEquals(4, expressions.size());
```

Comment:
Because reg exp `.*` by default does not matches EOL. So it did not matched single line annotations which ended with EOL. Now it matches them, because there is no useless EOL at the end